lib/active_admin/resource/attributes.rb

```diff
@@ -0,0 +1,44 @@
+module ActiveAdmin
+
+  class Resource
+    module Attributes
+
+      def default_attributes
+        resource_class.columns.each_with_object({}) do |c, attrs|
+          unless reject_col?(c)
```

Comment:
@deivid-rodriguez , feel free to create separate PR with changing default attributes you want to render and we will discuss what kind of columns we should keep and leave and what tests and features we should add to cover this.